pytorch/training/docker/1.11/py3/sdk2.4.0/Dockerfile.neuron.os_scan_allowlist.json

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
Can we review the packages in this file and remove if its been fixed by new sdk release, or check the job logs for the latest scan report